config/samples/dataplane_v1beta1_openstackdataplanenodeset_baremetal_with_ipam.yaml

```diff
@@ -9,8 +9,8 @@ spec:
     - name: ANSIBLE_ENABLE_TASK_DEBUGGER
       value: "True"
   services:
-    - download-cache
     - bootstrap
+    - download-cache
```

Comment:
why this?

download-cache should be the first service to be executed.

Reply:
I think we can actually drop the services list entirely from the samples that are just using the default list.

Reply:
i'm fine with a couple of different PRs

Reply:
Oh, I think @rebtoor's question is valid though. Can't we just leave the list unchanged for now? You said you hit a problem when you dropped the services list, but can we just leave it for now and clean it up in a later PR?